app/models/user.rb

```diff
@@ -4,9 +4,10 @@ class User < ActiveRecord::Base
             format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i }
   has_secure_password
   has_many :orders
+  has_many :businesses
   has_many :order_items, through: :orders
   has_many :items, through: :order_items
-  enum role: %w(default admin)
+  enum role: %w(default admin business_admin)
```

Comment:
Nit picky, but the list of roles should go in order of privilege. This should be 
```ruby
enum role: %w(default business_admin admin)
```

Then again, you can probably leave it alone, since we are really going to roll our own authentication, right?